AlphaWallet/UI/Views/AmountTextField.swift

```diff
@@ -34,26 +35,28 @@ class AmountTextField: UIControl {
     var ethCost: String {
         get {
             switch currentPair.left {
-            case .eth, .xDai:
+            case .cryptoCurrency(_):
```

Comment:
FYI. We can just use `case .cryptoCurrency:` if you don't need the associated values. (I switch and unused variable to `_` and then forget to remove it all the time. No biggie)